python/sdss_access/__init__.py

```diff
@@ -14,14 +14,19 @@
 # File logging can be started by calling log.start_file_logger(name).
 from .misc import log
 
+# check if posix-based operating system
+from os import name
+is_posix = ( name == "posix" )
+os_windows = not is_posix
```

Comment:
I think this is ok, but I'm just curious why both `is_posix` and `os_windows` are needed?

Reply:
is_posix is the real variable, os_windows is just for convenience (since os_windows = not is_posix).

Reply:
Michael and I agree that it would be simpler to drop os_windows and use not is_posix instead.  What do you think?

Reply:
It's up to you.  It's a bit redundant but I'm ok with leaving it in.  Sometimes it's a pain to always specify "not xxxx".  If you find that you're using `os_windows` a lot then I say leave it in.

Reply:
Actually, I think most if blocks would handle the is_posix case first anyway, with the else then handling windows, so lets proceed with dropping os_windows.